transcom/mymove
# B 21557 Main update price estimates on estimated weight update

## [B-21557](https://www13.v1host.com/USTRANSCOM38/TeamRoom.mvc/Show/848240)
## [INT PR](https://github.com/transcom/mymove/pull/14427)
## Summary

Updates the price estimates on service items where an estimated weight wasn't initially provided.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Create a move as an HHG shipment, or find an existing move that is already approved
2. As a SC add an HHG shipment to the move. but do not add an estimated weight
3. Verify on the move task order tab that the service items are added, leave the tab open
4. Open a separate tab and find the move as a prime user
5. update the new mto shipment within the move with an estimated weight
6. back on the SC tab, refresh the mto shipment tab, the prices should now show next to the service items
